python/sglang/srt/managers/scheduler.py

```diff
@@ -346,12 +354,17 @@ def event_loop_normal(self):
                         self.update_running_batch()
                         if not self.running_batch:
                             break
+                        if self.server_args.enable_dp_mla:
+                            self.check_dp_batch_running(batch)
```

Comment:
It seems the return value is not used.

Reply:
Yes, we just need a sync before the run_batch.